### DISCUSSION TITLE
Hi, I'm a novice, I once tried to use github. Since that time I got a problem : everytime i open one website that is in my favorite, the adress of the website is changed systematically to https://app.sindup.com/github-markdown-css. How can I delete defintely "/github-markdown-css" from the website adresse ? Thank for helpeing me

### DISCUSSION THREAD

**Discussion by pratarti** [community/community#121220] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, I'm a novice, I once tried to use github. Since that time I got a problem : everytime I open one website that is in my favorite, the adress of the website is changed systematically to https://app.sindup.com/github-markdown-css. How can I delete defintely "/github-markdown-css" from the website adresse ? 
Thank for help

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @pratarti ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by pratarti** [reply]
thanks for you help
